packages/graphql/tests/schema/aggregations.test.ts

```diff
@@ -734,6 +734,10 @@ describe("Aggregations", () => {
 
             input PostLikesConnectFieldInput {
               edge: LikesCreateInput
+              \\"\\"\\"
+              Whether or not to overwrite any matching relationship with the new properties.
+              \\"\\"\\"
+              overwrite: Boolean
```

Comment:
I don't know about you, but personally, I like the schema to explain the default behaviour, so should this definition possibly be:

```suggestion
              overwrite: Boolean! = true
```

Reply:
Agreed!!